src/js/angular/ttyg/controllers/ttyg-view.controller.js

```diff
@@ -276,6 +279,49 @@ function TTYGViewCtrl($rootScope, $scope, $http, $timeout, $translate, $uibModal
             });
     };
 
+    $scope.onCloneAgent = (agent) => {
+        let agentToClone = agent;
```

Comment:
What is the purpose of this assignment?
I see `onEditAgent` does it also.

Reply:
My bad. For the edit operation it isneeded because you can trigger the edit operation via the edit button for an agent in the right sidebar, then you have the `agent` parameter, or via the edit button in the toolbar above the chat panel in which case you are editing the currently selected agent and in this case the `agent` parameter is undefined.
In the case for the clone operation though, this is not needed, because you can clone only through the menu on each agent onthe right sidebar.

Reply:
This is documented in the jsdoc for the edit action handler also

Reply:
removed